README.md

```diff
@@ -59,6 +60,7 @@ pins (pushed in all the way) and metal threaded pins (sticking out) with plastic
 - [ ] Check that wipers function correctly and look for potential wiper dings on hood
 - [ ] Inspect every glass and make sure there are no scratches or cracks 
 - [ ] Check for trim scratches on windows due to a software bug that doesn't automatically lower the window when opening the door
+- [ ] Express Pickup: Check the charge of the vehicle. It comes from the factory at 50% but the local dealer should charge it to 100%.
```

Comment:
Hmm, I am not sure about this. Charging to 100% puts chemical stress on the battery. For a long life Li-Ion batteries should be kept charged around 50-60%. I'd have been pretty upset if I'd have picked up my car charged to 100%.

Reply:
@polymorphic Yup.  Good point.  What I mean to say is that, dealers sell new cars with a full tank of gas, and Tesla policy (according to the sales rep) is really no different except they they are not currently "filling" the vehicles currently because it takes too much time.  I took ownership at 48%.  It should be charged to 100% of what Tesla considers to be the appropriate "daily driving" amount (which is 90% full charge).